make/kube-configs

```diff
@@ -4,7 +4,16 @@ set -o errexit -o nounset
 
 GIT_ROOT=${GIT_ROOT:-$(git rev-parse --show-toplevel)}
 
+# TODO: We currently skip memory limits for development purposes
+
+. "${GIT_ROOT}/make/include/registry"
+
 cd "$GIT_ROOT"
 . ${GIT_ROOT}/.envrc
-fissile build kube -k kube/ --use-memory-limits=false \
-	    -D $(echo env/*.env | tr ' ' ',')
+fissile build kube \
+    --kube-output-dir="${GIT_ROOT}/kube" \
+    --defaults-file="$(echo env/*.env | tr ' ' ',')" \
+    --use-memory-limits=false \
+    --docker-registry "${IMAGE_REGISTRY}" \
```

Comment:
I see nothing in the UFR `.envrc` about images and registry, and what not.
The info comes from the include `registry` file.
My understanding is that Alex added this here because it is `fissile` now which tags generated images, and thus the `build kube` command has to have the same info to insert the proper refs into the generated yaml files.

And in the context of the integration this info overridden from SCF